### PR TITLE
Migrate to VS 2019 and maintain project

### PR DESCRIPTION
**Pull request by zvirja**
Maintain project by switching to the latest VS 2019 CI image. It required me to update FAKE, StyleCop and SourceLink, so I did additional maintenance of that.

Notice, checking the individual commits might help to more clearly see what exactly changed and why.

### CODE REVIEW

**Comment by zvirja** [comment]
I've created a [PR](https://github.com/fsharp/FAKE/pull/2527) to FAKE to fix the issue. If they publish soon, then I could just upgrade to the latest FAKE and remove the workaround.

**Comment by aivascu** [comment]
This looks awesome!

File: Src/AutoFixture/SpecimenBuilderNodeAdapterCollection.cs [hunk 1]
```diff
@@ -113,6 +113,8 @@ private ISpecimenBuilderNode AdaptedBuilderNode
         /// <seealso cref="SpecimenBuilderNodeAdapterCollection(ISpecimenBuilderNode, Func{ISpecimenBuilderNode, bool})" />
         public int IndexOf(ISpecimenBuilder item)
         {
+            if (item == null) throw new ArgumentNullException(nameof(item));
```
Comment (aivascu) [inline Src/AutoFixture/SpecimenBuilderNodeAdapterCollection.cs:116]: Will we need to update some tests for this change?

Comment (zvirja) [reply Src/AutoFixture/SpecimenBuilderNodeAdapterCollection.cs:116]: Good catch 👍  I didn't do deep analysis around it. The reason I added this change is that the newest analyzers are failing about not checking the value. So it was not possible to run the build after I updated the analyzers.

I updated all the other places. Don't think it should be covered with unit tests, as in reality I believe it's only being used by our internal code - it's a collection => graph facade we use for tree mutation.

Let me know if you are fine with that.

**Comment by zvirja** [comment]
@aivascu I updated the FAKE to latest (with my fix), please give it another look. Thanks!

**Comment by zvirja** [comment]
@aivascu Let me do the next release, so I can confirm that CI pipeline works as expected after all the changes. After that please go ahead and take care of subsequent releases.

Let me know if you are fine ;)

**Comment by aivascu** [comment]
Thank you for making the release @zvirja! 👍 

Of course, I'll take care of the next releases.
Hopefully you can still answer some questions if I am going to have any. 😅

**Pull request merged**